### ISSUE TITLE
Comfy defaults

### ISSUE THREAD

**Issue by slrgt** [YouTube-Enhancer/extension#512]
**Is your feature request related to a problem? Please describe.**
I've been recommending this extension to more and more people but they always struggle setting it up because theyre bad with technology so going into menus can be much for them unlike something like ublock origin which works once installed and the menus are only needed for advanced users

**Describe the solution you'd like**
comfortable defaults that instantly are applied and make sense to the user. Which would be button placements on the bottom of the player for Screenshot, Decrease Speed, and Increase Speed. Automatically enabled Pause Background Players, Automatic Theatre Mode, Track Watched Videos, Resume type Automatic, Playback Button Speed .25 :)

**Comment by VampireChicken12** [comment]
I can look into making default config presets but no promises. I think it would make the size of the extension grow a bit because of having to include multiple default options instead of just the one

**Comment by VampireChicken12** [comment]
The extension kind of comes in a "some configuration required by the user" state on purpose. I didn't want to force opinionated defaults on users, instead I want them to be able to easily change them.

**Comment by slrgt** [comment]
Opinionated defaults are good as long as the user can change them. I much prefer having defaults that add easily understood benefits to the user that they can then tweak to their preference than requiring to explain every option they probably want set. This way it enhances their experience instantly once installed

**Comment by VampireChicken12** [comment]
Yeah. The settings export does exist. You could configure the extension how your friends want then export settings then they can import.

**Comment by danielwerg** [comment]
I, on the contrary, preferred having all options disabled as onboarding experience, which allowed me to carefully inspect each option one by one. If this gets implemented can we have a turn off all feature/options button?
